src/mvs_eland/C0_data_processing.py

```diff
@@ -33,9 +33,10 @@
 - Read timeseries of assets and store into json (differ between one-column csv, multi-column csv)
 - Read timeseries for parameter of an asset, eg. efficiency
 - Parse list of inputs/outputs, eg. for chp
-- Define dso sinks, soures, transformer stations (this will be changed due to bug #119), also for peak demand pricing
+- Define dso sinks, sources, transformer stations (this will be changed due to bug #119), also for peak demand pricing
 - Add a source if a conversion object is connected to a new input_direction (bug #186)
 - Define all necessary energyBusses and add all assets that are connected to them specifically with asset name and label
+- Multiply `maximumCap` of non-dispatchable sources by max(timeseries(kWh/kWp)) to fix `optimizedAddCap` (see issue #446)
```

Comment:
```suggestion
- Multiply `maximumCap` of non-dispatchable sources by max(timeseries(kWh/kWp)) as the `maximumCap` is limiting the flow but we want to limit the installed capacity (see issue #446)
```

Reply:
Uh, yeah, I actually need to update this section as well for my changes. I was actually not quite sure if we need it after we have defined all these pytests...?

Reply:
I think a module description is always nice! I actually do read them when I want to get an overview over module.